src/main/java/com/meraphorce/dto/ResponseDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.meraphorce.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Getter
+@Setter
+public class ResponseDTO<T> {
+	private T object;
+	private String message;
+	private boolean status;
+
+}
```

Comment:
Add new line.
Ending a file with a newline (also known as a line break or an end-of-line (EOL) character) is a convention rooted in the definitions of Unix-based systems and is considered a best practice for several practical reasons in software development